.circleci/config.yml

```diff
@@ -182,6 +185,35 @@ jobs:
           path: frontend/reports/
       - store_artifacts:
           path: frontend/coverage/
+  dynamic_security_scan:
+    executor: machine-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Start up local server
+          command: |
+            docker-compose -f docker-compose-test.yml run --rm server yarn install --production=false
+            docker-compose -f docker-compose-test.yml run --rm server yarn --cwd frontend install --production=false
+            docker-compose -f docker-compose-test.yml run --rm server yarn build
+            docker-compose -f docker-compose-test.yml run --rm server yarn --cwd frontend run build
+            docker-compose -f docker-compose-test.yml up -d
+            docker-compose -f docker-compose-test.yml exec server yarn db:migrate
+      - run:
+          name: Pull OWASP ZAP docker image
+          command: docker pull owasp/zap2docker-weekly
+      - run:
+          name: Run OWASP ZAP
+          command: |
+            docker run \
```

Comment:
Would be nice to move this command into a command that can be ran locally (maybe a yarn command?) I don't think it would need to include all the `docker-compose` setup, just the `docker run` command